docs/lib/storage/fragments/android/upload.md

```diff
@@ -136,3 +137,60 @@ upload
 
 </amplify-block>
 </amplify-block-switcher>
+
+## Upload inputStream
+To upload to S3 from a data object, specify the key and the InputStream to be uploaded. 
+
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+private void uploadInputStream() {
+    InputStream exampleInputStream = new ByteArrayInputStream(RandomBytes.bytes());
```

Comment:
For the sample, can you generate how to obtain the `InputStream` from a `Uri`?  A `Uri` is what you get back from the Android file/photo picker, so this is the most common use case that customers will be needing.